vendor/google.golang.org/api/internal/creds.go

```diff
@@ -57,7 +57,7 @@ func baseCreds(ctx context.Context, ds *DialSettings) (*google.Credentials, erro
 	if len(cred.JSON) > 0 {
 		return credentialsFromJSON(ctx, cred.JSON, ds)
 	}
-	// For GAE and GCE, the JSON is empty so return the default credentials directly.
+	// For GAE and GCP, the JSON is empty so return the default credentials directly.
```

Comment:
Per internal chat I think we need to avoid changing things in `vendor/`.  This would be a patch for the upstream project, but as this is just a comment, I think we can just skip this.